frontend/src/metabase/common/components/AggregationPicker/AggregationPicker.tsx

```diff
@@ -122,6 +123,7 @@ export function AggregationPicker({
         name: t`Custom Expression`,
         items: [],
         icon: "sum",
+        type: "action",
```

Comment:
Custom expression button now has green text color

![image](https://github.com/metabase/metabase/assets/6830683/5d8efa3c-c159-46b0-bcb8-01fcdb1da168)

Reply:
This makes it consistent with the other buttons. But I agree that it is a bit much compared to the other buttons, which have a more toned down active color.

@AlessioLaiso should we remove the color for buttons in the dropdown altogether?

Reply:
We explored adding color along these lines but decided against it. This green on white isn't great for accessibility either. TL;DR here: Divider and chevron are a yes, green text is a no.